react/ui-components/src/atoms/TextInput.js

```
@@ -23,11 +21,7 @@ const TextInput = (props) => {
   const handleDate = (event) => {
     const { value } = event?.target;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)